npm/pkg/controlplane/controllers/v2/networkPolicyController.go

```diff
@@ -224,13 +224,13 @@ func (c *NetworkPolicyController) syncNetPol(key string) error {
 		return nil
 	}
 
-	cachedNetPolObj, netPolExists := c.rawNpMap[key]
+	cachedNetPolSpecObj, netPolExists := c.rawNpMap[key]
```

Comment:
I worried about `updating name and namespace `, but they are immutable.

Reply:
Yes and our Key has namespace and name in it.